docs/docs/core/strapi/entity-service.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Entity Service
```

Comment:
```suggestion
title: Entity Service
slug: /strapi/entity-service
```

This should give us the desired URL :)

Reply:
@joshuaellis currently put it in a strapi folder and also added a strapi section to the docs

Reply:
no, fine is both, this is how we're doing it everywhere else iirc